pkg/atlantis/README.md

```diff
@@ -0,0 +1,176 @@
+# `atlantis` integration
+
+## Atlantis Repo Config Generation
+
+`atmos` supports generating [Repo Level atlantis.yaml Config](https://www.runatlantis.io/docs/repo-level-atlantis-yaml.html) for `atmos` components
+and stacks.
+
+The following `atmos` commands will first generate the varfiles for all components in all stacks,
+then generate the `atlantis.yaml` repo config file:
+
+```bash
+  atmos terraform generate varfiles --file-template=varfiles/{tenant}-{environment}-{stage}-{component}.tfvars.json
+  atmos atlantis generate repo-config --config-template config-template-1 --project-template project-template-1 --workflow-template workflow-template-1
```

Comment:
```suggestion
atmos terraform generate varfiles --file-template=varfiles/{tenant}-{environment}-{stage}-{component}.tfvars.json
atmos atlantis generate repo-config --config-template config-template-1 --project-template project-template-1 --workflow-template workflow-template-1
```

Reply:
that's weird the `md fmt` adds 2 spaces to commands. Seems like a bug in their script.